src/helper.ts

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as gax from 'google-gax';
+import {Descriptors, ClientOptions} from 'google-gax';
+import * as path from 'path';
+import * as gapicConfig from './iam_policy_service_client_config.json';
+import {ProjectIdCallback} from 'google-auth-library';
+import * as protosTypes from '../protos/protos';
+const version = require('../../package.json').version;
```

Comment:
I think we have the flag set in `tsconfig` to allow directly importing this thing.  If not, let's use destructuring assignment here.

Reply:
Reasonable, previously it's like below, but `npm` copies `package.json` into `build/` folder. And this results in the abnormal tarball content when `npm pack`. (only includes package.json, readme ....., not `build/src`, `build/protos`)
```
import * as packagejson from '../package.json';
const version = packagejson.version;
```

Reply:
@xiaozhenliu-gg5 But still, even if we keep `require`, can we use destructuring as Justin suggests?

```js
const {version} = require('../../package.json');
```

Reply:
Just pointing out, this one is still unresolved as well :)

Reply:
Thanks for thre reminder, seems I marked as resolved. If any other comments are added, they won't stand out. :)